language/tools/move-cli/src/package/cli.rs

```diff
@@ -163,6 +166,12 @@ pub enum PackageCommand {
         /// Collect coverage information for later use with the various `package coverage` subcommands
         #[clap(long = "coverage")]
         compute_coverage: bool,
+
+        /// Use the EVM-based execution backend.
+        /// Does not work with --stackless.
+        #[cfg(feature = "evm-backend")]
+        #[structopt(long = "evm")]
```

Comment:
How can structopt still be here? We've been switched by a friendly soul to solely use clap 3, which AFAICT includes structopt.

Reply:
I double checked `move-cli`'s Cargo.toml and can confirm that we have indeed switched to clap 3. Maybe they kept `structopt` for backward compatibility reasons? Anyway, I'll get this fixed in a future PR.